bundles/org.openhab.io.hueemulation/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -51,9 +51,9 @@
 			<description>All items that are tagged with the given tags are ignore by the Hue Emulation Service. Use commas to separate multiple entries.</description>
 			<default>internal</default>
 		</parameter>
-		<parameter name="discoveryIp" type="text" required="false">
+		<parameter name="discoveryIps" type="text" required="false">
```

Comment:
I think you should change this back and just change the field name in the HueEmulationConfig.
You can keep the the description change though.

Reply:
But if I change it back then the requests to the backend once again will be PUT'ed to the backend as `discoveryIp`. Is this acceptable as this does not match well the README docs? However changing it here to `Ips` probably does not fill in the value for the users where it is already saved as `discoveryIp`. Probably ur right. Have to look into it in the evening for this change.

Reply:
Changing the config parameter name here would break backwards compatibility by forcing users to recreate their things. All you need to do is update the field name in the code and everything should work correctly for existing users. You don't need to keep the old field name in the config code since it was wrong to begin with and thus was never used.

Reply:
done